### PR TITLE
Rename to acorn-io/runtime

### PR DESCRIPTION
**Pull request by thedadams**

### CODE REVIEW

**Comment by thedadams** [comment]
This will merge after the repository is renamed.

**Review by g-linville — APPROVED** [review]

**Comment by tylerslaton** [comment]
Should this be `runtime`?

https://github.com/acorn-io/acorn/blob/e790cf834c6727999e58de3cbf757eaa93c292ee/.goreleaser.yml#L80C6-L80C6

**Comment by thedadams** [comment]
@tylerslaton Yes! Great catch. Both should be `runtime`.

File: docs/versioned_docs/version-0.3/10-home.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: /
 
 :::caution
 
-Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/acorn/discussions), or [Issues](https://github.com/acorn-io/acorn/issues)!
+Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/runtime/discussions), or [Issues](https://github.com/acorn-io/runtime/issues)!
```
Comment (njhale) [inline docs/versioned_docs/version-0.3/10-home.md:8]: Do we want to update docs for older releases? With the redirects, they shouldn't break.

**Review by tylerslaton — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	acorn "github.com/acorn-io/acorn/pkg/cli"
+	acorn "github.com/acorn-io/runtime/pkg/cli"
```
Comment (njhale) [inline main.go:4]:
```suggestion
	runtime "github.com/acorn-io/runtime/pkg/cli"
```

File: pkg/apis/internal.admin.acorn.io/v1/scheme.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package v1
 
 import (
-	internal_admin_acorn_io "github.com/acorn-io/acorn/pkg/apis/internal.admin.acorn.io"
+	internal_admin_acorn_io "github.com/acorn-io/runtime/pkg/apis/internal.admin.acorn.io"
```
Comment (njhale) [inline pkg/apis/internal.admin.acorn.io/v1/scheme.go:4]: nit(not for this PR): interesting snake_case

**Review by njhale — APPROVED** [review]
LGTM -- let's see what breaks.

File: docs/versioned_docs/version-0.3/10-home.md (hunk 1 above)
Comment (thedadams) [reply docs/versioned_docs/version-0.3/10-home.md:8]: I am open to not doing it. I also thought that the updating it wouldn't break anything either.

Do you have a preference for not doing this?

File: main.go (hunk 2 above)
Comment (thedadams) [reply main.go:4]: This seems to be referring to the actual `acorn` CLI command, so I would be inclined to keep it as is. Would you still feel better about changing it?

File: docs/versioned_docs/version-0.3/10-home.md (hunk 1 above)
Comment (njhale) [reply docs/versioned_docs/version-0.3/10-home.md:8]: I did, but on second thought, we'll eventually remove these, and I don't expect users to be poking around in such old docs anyway.

So, I'm fine with merging as-is, old doc changes included.

File: main.go (hunk 2 above)
Comment (njhale) [reply main.go:4]: good point. I'm fine with it as is.

**Pull request merged**